.travis.yml

```diff
@@ -1,16 +1,21 @@
-sudo: false
+sudo: required # due to manual install of make 4.1
```

Comment:
You're not running in their docker container when you have sudo:required though. I couldn't find a way to get make 4.1 installed with sudo:false, otherwise I would've stayed with that. I'd hoped to sneak a modern make in via sources:debian-sid, but that *really* didn't work, and there is no xenial support anywhere to be found.

If you find a way, feel free to shift things around here. You can see the stuff I've already tried in the original branch.